src/main/resources/plugin.yml

```diff
@@ -1,6 +1,6 @@
 name: ${project.name}
 main: ${project.groupId}.${project.artifactId}.${project.name}
 version: ${project.version}
-api-version: "1.15"
+api-version: "1.18"
```

Comment:
This forces the users of the plugin to be running 1.18 when it can still run just fine on the version it was originally written against - the packet format has not changed. I get that 1.18 is the majority market share, but I don't see a reason to break backwards compatibility in this case. This is also why I didn't bother updating dependencies.

Reply:
I guess we may as well update and go with recent Java constructs though, it'll clean up the code a touch.